The job **`i686-msvc`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/11872505915/job/33086304883) [(plain)](https://github.com/rust-lang-ci/rust/commit/f6afd1b701a47851b89b72f64e46ac3b50666524/checks/33086304883/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
failures:

---- [debuginfo-cdb] tests\debuginfo\numeric-types.rs stdout ----

error: check directive(s) from `C:\a\rust\rust\tests\debuginfo\numeric-types.rs` not found in debugger output. errors:
    (numeric-types.rs:86) `    [+0x000] __0              : 60 [Type: __int64]`
    (numeric-types.rs:110) `    [+0x000] __0              : 0x78 [Type: unsigned __int64]`
the following subset of check directive(s) was found successfully:
    (numeric-types.rs:17) `nz_i8            : 11 [Type: core::num::nonzero::NonZero<i8>]`
    (numeric-types.rs:18) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<i8>]`
    (numeric-types.rs:21) `nz_i16           : 22 [Type: core::num::nonzero::NonZero<i16>]`
    (numeric-types.rs:22) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<i16>]`
    (numeric-types.rs:25) `nz_i32           : 33 [Type: core::num::nonzero::NonZero<i32>]`
    (numeric-types.rs:26) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<i32>]`
    (numeric-types.rs:29) `nz_i64           : 44 [Type: core::num::nonzero::NonZero<i64>]`
    (numeric-types.rs:30) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<i64>]`
    (numeric-types.rs:34) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<i128>]`
    (numeric-types.rs:37) `nz_isize         : 66 [Type: core::num::nonzero::NonZero<isize>]`
    (numeric-types.rs:38) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<isize>]`
    (numeric-types.rs:41) `nz_u8            : 0x4d [Type: core::num::nonzero::NonZero<u8>]`
    (numeric-types.rs:42) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<u8>]`
    (numeric-types.rs:45) `nz_u16           : 0x58 [Type: core::num::nonzero::NonZero<u16>]`
    (numeric-types.rs:46) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<u16>]`
    (numeric-types.rs:49) `nz_u32           : 0x63 [Type: core::num::nonzero::NonZero<u32>]`
    (numeric-types.rs:50) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<u32>]`
    (numeric-types.rs:53) `nz_u64           : 0x64 [Type: core::num::nonzero::NonZero<u64>]`
    (numeric-types.rs:54) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<u64>]`
    (numeric-types.rs:57) `nz_u128          : 111 [Type: core::num::nonzero::NonZero<u128>]`
    (numeric-types.rs:58) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<u128>]`
    (numeric-types.rs:61) `nz_usize         : 0x7a [Type: core::num::nonzero::NonZero<usize>]`
    (numeric-types.rs:62) `    [<Raw View>]     [Type: core::num::nonzero::NonZero<usize>]`
    (numeric-types.rs:65) `w_i8             : 10 [Type: core::num::wrapping::Wrapping<i8>]`
    (numeric-types.rs:66) `    [+0x000] __0              : 10 [Type: char]`
    (numeric-types.rs:69) `w_i16            : 20 [Type: core::num::wrapping::Wrapping<i16>]`
    (numeric-types.rs:70) `    [+0x000] __0              : 20 [Type: short]`
    (numeric-types.rs:73) `w_i32            : 30 [Type: core::num::wrapping::Wrapping<i32>]`
    (numeric-types.rs:74) `    [+0x000] __0              : 30 [Type: int]`
    (numeric-types.rs:77) `w_i64            : 40 [Type: core::num::wrapping::Wrapping<i64>]`
    (numeric-types.rs:78) `    [+0x000] __0              : 40 [Type: __int64]`
    (numeric-types.rs:81) `w_i128           : 50 [Type: core::num::wrapping::Wrapping<i128>]`
    (numeric-types.rs:82) `    [+0x000] __0              : 50 [Type: i128]`
    (numeric-types.rs:85) `w_isize          : 60 [Type: core::num::wrapping::Wrapping<isize>]`
    (numeric-types.rs:89) `w_u8             : 0x46 [Type: core::num::wrapping::Wrapping<u8>]`
    (numeric-types.rs:90) `    [+0x000] __0              : 0x46 [Type: unsigned char]`
    (numeric-types.rs:93) `w_u16            : 0x50 [Type: core::num::wrapping::Wrapping<u16>]`
    (numeric-types.rs:94) `    [+0x000] __0              : 0x50 [Type: unsigned short]`
    (numeric-types.rs:97) `w_u32            : 0x5a [Type: core::num::wrapping::Wrapping<u32>]`
    (numeric-types.rs:98) `    [+0x000] __0              : 0x5a [Type: unsigned int]`
    (numeric-types.rs:101) `w_u64            : 0x64 [Type: core::num::wrapping::Wrapping<u64>]`
    (numeric-types.rs:102) `    [+0x000] __0              : 0x64 [Type: unsigned __int64]`
    (numeric-types.rs:105) `w_u128           : 110 [Type: core::num::wrapping::Wrapping<u128>]`
    (numeric-types.rs:106) `    [+0x000] __0              : 110 [Type: u128]`
    (numeric-types.rs:109) `w_usize          : 0x78 [Type: core::num::wrapping::Wrapping<usize>]`
    (numeric-types.rs:113) `a_bool_t         : true [Type: core::sync::atomic::AtomicBool]`
    (numeric-types.rs:114) `    [+0x000] v                : 0x1 [Type: core::cell::UnsafeCell<u8>]`
    (numeric-types.rs:117) `a_bool_f         : false [Type: core::sync::atomic::AtomicBool]`
    (numeric-types.rs:118) `    [+0x000] v                : 0x0 [Type: core::cell::UnsafeCell<u8>]`
    (numeric-types.rs:121) `a_i8             : 2 [Type: core::sync::atomic::AtomicI8]`
    (numeric-types.rs:122) `    [+0x000] v                : 2 [Type: core::cell::UnsafeCell<i8>]`
    (numeric-types.rs:125) `a_i16            : 4 [Type: core::sync::atomic::AtomicI16]`
    (numeric-types.rs:126) `    [+0x000] v                : 4 [Type: core::cell::UnsafeCell<i16>]`
    (numeric-types.rs:129) `a_i32            : 8 [Type: core::sync::atomic::AtomicI32]`
    (numeric-types.rs:130) `    [+0x000] v                : 8 [Type: core::cell::UnsafeCell<i32>]`
    (numeric-types.rs:133) `a_i64            : 16 [Type: core::sync::atomic::AtomicI64]`
    (numeric-types.rs:134) `    [+0x000] v                : 16 [Type: core::cell::UnsafeCell<i64>]`
    (numeric-types.rs:137) `a_isize          : 32 [Type: core::sync::atomic::AtomicIsize]`
    (numeric-types.rs:138) `    [+0x000] v                : 32 [Type: core::cell::UnsafeCell<isize>]`
    (numeric-types.rs:141) `a_u8             : 0x40 [Type: core::sync::atomic::AtomicU8]`
    (numeric-types.rs:142) `    [+0x000] v                : 0x40 [Type: core::cell::UnsafeCell<u8>]`
    (numeric-types.rs:145) `a_u16            : 0x80 [Type: core::sync::atomic::AtomicU16]`
    (numeric-types.rs:146) `    [+0x000] v                : 0x80 [Type: core::cell::UnsafeCell<u16>]`
    (numeric-types.rs:149) `a_u32            : 0x100 [Type: core::sync::atomic::AtomicU32]`
    (numeric-types.rs:150) `    [+0x000] v                : 0x100 [Type: core::cell::UnsafeCell<u32>]`
    (numeric-types.rs:153) `a_u64            : 0x200 [Type: core::sync::atomic::AtomicU64]`
    (numeric-types.rs:154) `    [+0x000] v                : 0x200 [Type: core::cell::UnsafeCell<u64>]`
    (numeric-types.rs:157) `a_usize          : 0x400 [Type: core::sync::atomic::AtomicUsize]`
    (numeric-types.rs:158) `    [+0x000] v                : 0x400 [Type: core::cell::UnsafeCell<usize>]`
status: exit code: 0
command: PATH="C:\a\rust\rust\build\i686-pc-windows-msvc\stage2\lib\rustlib\i686-pc-windows-msvc\lib;C:\Program Files (x86)\Windows Kits\10\bin\x64;C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.42.34433\bin\HostX64\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.42.34433\bin\HostX64\x86;C:\a\rust\rust\build\i686-pc-windows-msvc\stage0-bootstrap-tools\i686-pc-windows-msvc\release\deps;C:\a\rust\rust\build\i686-pc-windows-msvc\stage0\bin;C:\Program Files\PowerShell\7;C:\a\_temp\msys64\mingw32\bin;C:\a\_temp\msys64\usr\local\bin;C:\a\_temp\msys64\usr\bin;C:\a\_temp\msys64\usr\bin;C:\a\rust\rust\ninja;C:\a\rust\rust\sccache;C:\a\_temp\setup-msys2;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS;C:\tools\zstd;C:\Program Files\Mercurial;C:\hostedtoolcache\windows\stack\3.1.1\x64;C:\cabal\bin;C:\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.4.2\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.13\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.7\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.432-6\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files\dotnet;C:\Program Files\PowerShell\7;C:\Program Files\Microsoft\Web Platform Installer;C:\Program Files\TortoiseSVN\bin;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn;C:\Program Files\Microsoft SQL Server\150\Tools\Binn;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn;C:\Program Files\Microsoft SQL Server\140\DTS\Binn;C:\Program Files\Microsoft SQL Server\150\DTS\Binn;C:\Program Files\Microsoft SQL Server\160\DTS\Binn;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI;C:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2;C:\Program Files\Amazon\SessionManagerPlugin\bin;C:\Program Files\Amazon\AWSSAMCLI\bin;C:\Program Files\Microsoft SQL Server\130\Tools\Binn;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps;C:\a\_temp\msys64\usr\bin\site_perl;C:\a\_temp\msys64\usr\bin\vendor_perl;C:\a\_temp\msys64\usr\bin\core_perl" "C:\\Program Files (x86)\\Windows Kits\\10\\Debuggers\\x86\\cdb.exe" "-lines" "-cf" "C:\\a\\rust\\rust\\build\\i686-pc-windows-msvc\\test\\debuginfo\\numeric-types.cdb\\numeric-types.debugger.script" "C:\\a\\rust\\rust\\build\\i686-pc-windows-msvc\\test\\debuginfo\\numeric-types.cdb\\a.exe"


************* Preparing the environment for Debugger Extensions Gallery repositories **************
   ExtensionRepository : Implicit
   UseExperimentalFeatureForNugetShare : true
   AllowNugetExeUpdate : true
   NonInteractiveNuget : true
   AllowNugetMSCredentialProviderInstall : true
   AllowParallelInitializationOfLocalRepositories : true

   EnableRedirectToV8JsProvider : false
   -- Configuring repositories
      ----> Repository : LocalInstalled, Enabled: true
      ----> Repository : UserExtensions, Enabled: true


>>>>>>>>>>>>> Preparing the environment for Debugger Extensions Gallery repositories completed, duration 0.000 seconds

************* Waiting for Debugger Extensions Gallery to Initialize **************

>>>>>>>>>>>>> Waiting for Debugger Extensions Gallery to Initialize completed, duration 0.031 seconds
   ----> Repository : UserExtensions, Enabled: true, Packages count: 0
   ----> Repository : LocalInstalled, Enabled: true, Packages count: 29
Microsoft (R) Windows Debugger Version 10.0.26100.2161 X86
Copyright (c) Microsoft Corporation. All rights reserved.


CommandLine: C:\a\rust\rust\build\i686-pc-windows-msvc\test\debuginfo\numeric-types.cdb\a.exe

************* Path validation summary **************
Response                         Time (ms)     Location
Deferred                                       srv*
Symbol search path is: srv*
Executable search path is: 
ModLoad: 00150000 00156000   a.exe   
ModLoad: 77d80000 77f29000   ntdll.dll
ModLoad: 76ee0000 76fd0000   C:\Windows\SysWOW64\KERNEL32.DLL
ModLoad: 769a0000 76bf1000   C:\Windows\SysWOW64\KERNELBASE.dll
ModLoad: 75240000 752e2000   C:\Windows\SysWOW64\apphelp.dll
ModLoad: 76880000 76993000   C:\Windows\SysWOW64\ucrtbase.dll
ModLoad: 754f0000 75505000   C:\Windows\SysWOW64\VCRUNTIME140.dll
ModLoad: 73cc0000 745ef000   C:\a\rust\rust\build\i686-pc-windows-msvc\stage2\lib\rustlib\i686-pc-windows-msvc\lib\std-183ad74c54f5b709.dll
ModLoad: 76c00000 76c65000   C:\Windows\SysWOW64\WS2_32.dll
ModLoad: 75470000 75497000   C:\Windows\SysWOW64\USERENV.dll
ModLoad: 775a0000 7765a000   C:\Windows\SysWOW64\RPCRT4.dll
ModLoad: 01170000 0122a000   C:\Windows\SysWOW64\RPCRT4.dll
ModLoad: 761b0000 76215000   C:\Windows\SysWOW64\bcryptprimitives.dll
(1c60.19f0): Break instruction exception - code 80000003 (first chance)
eax=00000000 ebx=009c2000 ecx=16d20000 edx=00000000 esi=ffffffff edi=001500f0
eip=77e35d01 esp=007af264 ebp=007af290 iopl=0         nv up ei pl zr na pe nc
cs=0023  ss=002b  ds=002b  es=002b  fs=0053  gs=002b             efl=00000246
ntdll!LdrpDoDebuggerBreak+0x2b:
0:000> version
0:000> version
Windows 10 Version 20348 MP (8 procs) Free x86 compatible
Product: Server, suite: TerminalServer DataCenter SingleUserTS
Edition build lab: 20348.1.amd64fre.fe_release.210507-1500
Debug session time: Sat Nov 16 20:05:35.998 2024 (UTC + 0:00)
System Uptime: 0 days 1:06:52.612
Process Uptime: 0 days 0:00:00.076
  Kernel time: 0 days 0:00:00.015
  User time: 0 days 0:00:00.000
Live user mode: <Local>
Microsoft (R) Windows Debugger Version 10.0.26100.2161 X86
Copyright (c) Microsoft Corporation. All rights reserved.


command line: '"C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\cdb.exe" -lines -cf C:\a\rust\rust\build\i686-pc-windows-msvc\test\debuginfo\numeric-types.cdb\numeric-types.debugger.script C:\a\rust\rust\build\i686-pc-windows-msvc\test\debuginfo\numeric-types.cdb\a.exe'  Debugger Process 0xA5C 
dbgeng:  image 10.0.26100.2161, 
        [path: C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\dbgeng.dll]
dbghelp: image 10.0.26100.2161, 
        [path: C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\dbghelp.dll]
        DIA version: 33140
Extension DLL search Path:
    C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\WINXP;C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\winext;C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\winext\arcade;C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\pri;C:\Program Files (x86)\Windows Kits\10\Debuggers\x86;C:\Users\runneradmin\AppData\Local\Dbg\EngineExtensions32;C:\Program Files (x86)\Windows Kits\10\Debuggers\x86;C:\a\rust\rust\build\i686-pc-windows-msvc\stage2\lib\rustlib\i686-pc-windows-msvc\lib;C:\Program Files (x86)\Windows Kits\10\bin\x64;C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.42.34433\bin\HostX64\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.42.34433\bin\HostX64\x86;C:\a\rust\rust\build\i686-pc-windows-msvc\stage0-bootstrap-tools\i686-pc-windows-msvc\release\deps;C:\a\rust\rust\build\i686-pc-windows-msvc\stage0\bin;C:\Program Files\PowerShell\7;C:\a\_temp\msys64\mingw32\bin;C:\a\_temp\msys64\usr\local\bin;C:\a\_temp\msys64\usr\bin;C:\a\_temp\msys64\usr\bin;C:\a\rust\rust\ninja;C:\a\rust\rust\sccache;C:\a\_temp\setup-msys2;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS;C:\tools\zstd;C:\Program Files\Mercurial;C:\hostedtoolcache\windows\stack\3.1.1\x64;C:\cabal\bin;C:\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.4.2\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.13\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.7\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.432-6\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files\dotnet;C:\Program Files\PowerShell\7;C:\Program Files\Microsoft\Web Platform Installer;C:\Program Files\TortoiseSVN\bin;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn;C:\Program Files\Microsoft SQL Server\150\Tools\Binn;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn;C:\Program Files\Microsoft SQL Server\140\DTS\Binn;C:\Program Files\Microsoft SQL Server\150\DTS\Binn;C:\Program Files\Microsoft SQL Server\160\DTS\Binn;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI;C:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2;C:\Program Files\Amazon\SessionManagerPlugin\bin;C:\Program Files\Amazon\AWSSAMCLI\bin;C:\Program Files\Microsoft SQL Server\130\Tools\Binn;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps;C:\a\_temp\msys64\usr\bin\site_perl;C:\a\_temp\msys64\usr\bin\vendor_perl;C:\a\_temp\msys64\usr\bin\core_perl
Extension DLL chain:
    wow64exts: image 10.0.26100.2161, API 1.0.0, 
        [path: C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\WINXP\wow64exts.dll]
    dbghelp: image 10.0.26100.2161, API 10.0.6, 
        [path: C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\dbghelp.dll]
    exts: image 10.0.26100.2161, API 1.0.0, 
        [path: C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\WINXP\exts.dll]
    uext: image 10.0.26100.2161, API 1.0.0, 
        [path: C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\winext\uext.dll]
    ntsdexts: image 10.0.26100.2161, API 1.0.0, 
        [path: C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\WINXP\ntsdexts.dll]
WOW64 extensions loaded
0:000> .nvlist
Loaded NatVis Files:
    <None Loaded>
0:000> bp `numeric-types.rs:290`
*** WARNING: Unable to verify checksum for a.exe
0:000>  g
Breakpoint 0 hit
eax=00000400 ebx=00000001 ecx=00000400 edx=00000200 esi=00000000 edi=007af984
eip=00151afb esp=007af6b0 ebp=007af8b0 iopl=0         nv up ei pl nz na pe nc
cs=0023  ss=002b  ds=002b  es=002b  fs=0053  gs=002b             efl=00000206
a!numeric_types::main+0x74b:
00151afb e810000000      call    a!numeric_types::zzz (00151b10)
0:000>  dx nz_i8
nz_i8            : 11 [Type: core::num::nonzero::NonZero<i8>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<i8>]
0:000>  dx nz_i16
nz_i16           : 22 [Type: core::num::nonzero::NonZero<i16>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<i16>]
0:000>  dx nz_i32
nz_i32           : 33 [Type: core::num::nonzero::NonZero<i32>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<i32>]
0:000>  dx nz_i64
nz_i64           : 44 [Type: core::num::nonzero::NonZero<i64>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<i64>]
0:000>  dx nz_i128
nz_i128          : 55 [Type: core::num::nonzero::NonZero<i128>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<i128>]
0:000>  dx nz_isize
nz_isize         : 66 [Type: core::num::nonzero::NonZero<isize>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<isize>]
0:000>  dx nz_u8
nz_u8            : 0x4d [Type: core::num::nonzero::NonZero<u8>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<u8>]
0:000>  dx nz_u16
nz_u16           : 0x58 [Type: core::num::nonzero::NonZero<u16>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<u16>]
0:000>  dx nz_u32
nz_u32           : 0x63 [Type: core::num::nonzero::NonZero<u32>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<u32>]
0:000>  dx nz_u64
nz_u64           : 0x64 [Type: core::num::nonzero::NonZero<u64>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<u64>]
0:000>  dx nz_u128
nz_u128          : 111 [Type: core::num::nonzero::NonZero<u128>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<u128>]
0:000>  dx nz_usize
nz_usize         : 0x7a [Type: core::num::nonzero::NonZero<usize>]
    [<Raw View>]     [Type: core::num::nonzero::NonZero<usize>]
0:000>  dx w_i8
w_i8             : 10 [Type: core::num::wrapping::Wrapping<i8>]
    [+0x000] __0              : 10 [Type: char]
0:000>  dx w_i16
w_i16            : 20 [Type: core::num::wrapping::Wrapping<i16>]
    [+0x000] __0              : 20 [Type: short]
0:000>  dx w_i32
w_i32            : 30 [Type: core::num::wrapping::Wrapping<i32>]
    [+0x000] __0              : 30 [Type: int]
0:000>  dx w_i64
w_i64            : 40 [Type: core::num::wrapping::Wrapping<i64>]
    [+0x000] __0              : 40 [Type: __int64]
0:000>  dx w_i128
w_i128           : 50 [Type: core::num::wrapping::Wrapping<i128>]
    [+0x000] __0              : 50 [Type: i128]
0:000>  dx w_isize
w_isize          : 60 [Type: core::num::wrapping::Wrapping<isize>]
    [+0x000] __0              : 60 [Type: int]
0:000>  dx w_u8
w_u8             : 0x46 [Type: core::num::wrapping::Wrapping<u8>]
    [+0x000] __0              : 0x46 [Type: unsigned char]
0:000>  dx w_u16
w_u16            : 0x50 [Type: core::num::wrapping::Wrapping<u16>]
    [+0x000] __0              : 0x50 [Type: unsigned short]
0:000>  dx w_u32
w_u32            : 0x5a [Type: core::num::wrapping::Wrapping<u32>]
    [+0x000] __0              : 0x5a [Type: unsigned int]
0:000>  dx w_u64
w_u64            : 0x64 [Type: core::num::wrapping::Wrapping<u64>]
    [+0x000] __0              : 0x64 [Type: unsigned __int64]
0:000>  dx w_u128
w_u128           : 110 [Type: core::num::wrapping::Wrapping<u128>]
    [+0x000] __0              : 110 [Type: u128]
0:000>  dx w_usize
w_usize          : 0x78 [Type: core::num::wrapping::Wrapping<usize>]
    [+0x000] __0              : 0x78 [Type: unsigned int]
0:000>  dx a_bool_t
a_bool_t         : true [Type: core::sync::atomic::AtomicBool]
    [+0x000] v                : 0x1 [Type: core::cell::UnsafeCell<u8>]
0:000>  dx a_bool_f
a_bool_f         : false [Type: core::sync::atomic::AtomicBool]
    [+0x000] v                : 0x0 [Type: core::cell::UnsafeCell<u8>]
0:000>  dx a_i8
a_i8             : 2 [Type: core::sync::atomic::AtomicI8]
    [+0x000] v                : 2 [Type: core::cell::UnsafeCell<i8>]
0:000>  dx a_i16
a_i16            : 4 [Type: core::sync::atomic::AtomicI16]
    [+0x000] v                : 4 [Type: core::cell::UnsafeCell<i16>]
0:000>  dx a_i32
a_i32            : 8 [Type: core::sync::atomic::AtomicI32]
    [+0x000] v                : 8 [Type: core::cell::UnsafeCell<i32>]
0:000>  dx a_i64
a_i64            : 16 [Type: core::sync::atomic::AtomicI64]
    [+0x000] v                : 16 [Type: core::cell::UnsafeCell<i64>]
0:000>  dx a_isize
a_isize          : 32 [Type: core::sync::atomic::AtomicIsize]
    [+0x000] v                : 32 [Type: core::cell::UnsafeCell<isize>]
0:000>  dx a_u8
a_u8             : 0x40 [Type: core::sync::atomic::AtomicU8]
    [+0x000] v                : 0x40 [Type: core::cell::UnsafeCell<u8>]
0:000>  dx a_u16
a_u16            : 0x80 [Type: core::sync::atomic::AtomicU16]
    [+0x000] v                : 0x80 [Type: core::cell::UnsafeCell<u16>]
0:000>  dx a_u32
a_u32            : 0x100 [Type: core::sync::atomic::AtomicU32]
    [+0x000] v                : 0x100 [Type: core::cell::UnsafeCell<u32>]
0:000>  dx a_u64
a_u64            : 0x200 [Type: core::sync::atomic::AtomicU64]
    [+0x000] v                : 0x200 [Type: core::cell::UnsafeCell<u64>]
0:000>  dx a_usize
a_usize          : 0x400 [Type: core::sync::atomic::AtomicUsize]
    [+0x000] v                : 0x400 [Type: core::cell::UnsafeCell<usize>]
0:000> qq
quit:
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\atlmfc.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\ObjectiveC.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\concurrency.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\cpp_rest.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\stl.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\Windows.Data.Json.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\Windows.Devices.Geolocation.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\Windows.Devices.Sensors.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\Windows.Media.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\windows.natvis'
NatVis script unloaded from 'C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\Visualizers\winrt.natvis'
stderr: none




failures:
    [debuginfo-cdb] tests\debuginfo\numeric-types.rs

test result: FAILED. 158 passed; 1 failed; 3 ignored; 0 measured; 0 filtered out; finished in 21.81s

Some tests failed in compiletest suite=debuginfo mode=debuginfo host=i686-pc-windows-msvc target=i686-pc-windows-msvc
Build completed unsuccessfully in 0:51:41
make: *** [Makefile:106: ci-msvc-ps1] Error 1
  network time: Sat, 16 Nov 2024 20:05:43 GMT
##[error]Process completed with exit code 2.
Post job cleanup.
[command]"C:\Program Files\Git\bin\git.exe" version
```

</details>

--- a/tests/debuginfo/numeric-types.rs
+++ b/tests/debuginfo/numeric-types.rs
@@ -3,8 +3,11 @@
 //@ ignore-windows-gnu: #128981
 
 // Note: u128 visualization was not supported in 10.0.22621.3233 but was fixed in 10.0.26100.2161.
-// FIXME(jieyouxu): triple-check if this annotation works properly in CI, because it seems to
-// ignore the test locally for me.
+
+// FIXME(#133107): this is temporarily marked as `only-64bit` because of course 32-bit msvc has
+// a different integer width and thus underlying integer type display. Only marked as such to
+// unblock the tree.
+//@ only-64bit
 //@ min-cdb-version: 10.0.26100.2161
 
 // Tests the visualizations for `NonZero<T>`, `Wrapping<T>` and

